hera_cal/tests/test_delay_filter.py

```diff
@@ -50,15 +50,16 @@ def test_run_delay_filter(self):
             np.testing.assert_array_equal(dfil.clean_model[k][0, :], np.zeros_like(dfil.clean_resid[k][0, :]))
             np.testing.assert_array_equal(dfil.clean_resid[k][0, :], np.zeros_like(dfil.clean_resid[k][0, :]))
 
-    def test_write_filtered_data(self):
+    def test_write_filtered_data(self, tmpdir):
```

Comment:
What's the advantage to `tmpdir` vs. `DATA_PATH/test_output`?

Reply:
tmpdir gets cleaned up after the test?